packages/flutter/test/widgets/list_view_test.dart

```diff
@@ -86,12 +140,12 @@ void main() {
           itemExtent: 200.0,
           children: new List<Widget>.generate(20, (int i) {
             return new Builder(
-              builder: (BuildContext context) {
-                log.add(i);
-                return new Container(
-                  child: new Text('$i'),
-                );
-              }
+                builder: (BuildContext context) {
+                  log.add(i);
+                  return new Container(
+                    child: new Text('$i'),
+                  );
+                }
```

Comment:
Revert the whitespace-only change? Adding a comma after `}` would be the correct Flutter-style.